node-commons/src/main/java/org/apache/iotdb/commons/conf/CommonConfig.java

```diff
@@ -128,6 +128,8 @@ public class CommonConfig {
   /** Ip and port of target ML node. */
   private TEndPoint targetMLNodeEndPoint = new TEndPoint("127.0.0.1", 10810);
 
+  private String timestampPrecision = "ms";
```

Comment:
Better move this parameter into iotdb-common.properties. Because this parameter is global consistent.

Reply:
My mistake